<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1091</width>
    <height>807</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1091</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>1021</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt; font-weight:600;&quot;&gt;Composici칩n de Portfolio y Simulaci칩n de Crecimiento&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tablaDatos">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>470</y>
     <width>401</width>
     <height>291</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>401</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graph">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>63</y>
     <width>771</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_startSimulation">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Empezar simulaci칩n</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graph_bar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>470</y>
     <width>451</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="desplegable_year">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2023</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2022</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>A침o</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Riesgo</string>
   </property>
  </widget>
  <widget class="QComboBox" name="desplegable_riesgo">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Moderado</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Conservador</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Alto</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="coins">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>171</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="capital">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Capital</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
